e2e/utils/command-utils.ts

```diff
@@ -332,7 +332,7 @@ export function runCLI(
     const logs = execSync(
       `${pm.runNxSilent} ${command} ${
         opts.verbose ?? isVerboseE2ERun() ? ' --verbose' : ''
-      }`,
+      } 2>&1`,
```

Comment:
Nice catch

Reply:
Could we have this as an additional flag to the call? (can be done in a follow up PR)

Reply:
Don't we generally want to keep all logs, though? I mean, when would we not want to show the console.error?

Reply:
Usually console.error is accompanied by an actual error so this is a special case.

Reply:
It's probably fine, since additional logs (errors) can just be ignored in the returned output. If it becomes and issue for some reason later we can add the option.

Reply:
Some e2e tests where failing, so I added the option! :)